modules/install-sno-requirements-for-installing-on-a-single-node.adoc

```diff
@@ -7,7 +7,7 @@
 
 Installing {product-title} on a single node alleviates some of the requirements of high availability and large scale clusters. However, you must address the following requirements:
 
-* *Administration node:* You must have an administration node or laptop to access link:https://console.redhat.com/openshift/assisted-installer/clusters[Install OpenShift with the Assisted Installer], to specify the cluster name, to create the USB boot drive, and to monitor the installation.
+* *Administration node:* You must have an administration node or laptop to prepare the ISO, to create the USB boot drive, and to monitor the installation.
```

Comment:
I'm not sure node is the correct term here either. Does "node" connotate a host that is installed in the cluster? Would host be better? 

```suggestion
* *Administration host:* You must have a computer to prepare the ISO, to create the USB boot drive, and to monitor the installation.
```

Reply:
Switched to host.